components/cycles/CycleOrder.jsx

```diff
@@ -0,0 +1,131 @@
+import { View } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import { IconButton } from 'react-native-paper';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+
+const CycleTitle = styled.Text`
+   color: #FFFFFF;
+   font-size: 24px;
+   font-family: 'Montserrat_500Medium';
+   textAlign: left;
+   paddingLeft: 15px;
+   paddingTop: 5px;
+`;
+
+const SubTitle = styled(CycleTitle)`
+    font-size: 16px;
+    paddingBottom: 5px;
+    font-family: 'Montserrat_600SemiBold';
+`;
+
+const WorkoutViewText = styled.Text`
+    color: #FFFFFF;
+    font-family: 'Montserrat_500Medium';
+    font-size: 12px;
+    paddingTop: 2px;
+    paddingBottom: 4px;
+    width: 33%;
+    textAlign: center;
+`;
+
+const RowHeader = styled.View`
+    flexDirection: row;
+    alignContent: center;
+    font-family: 'Montserrat_500Medium';
+    justifyContent: space-evenly;
+    width:100%;
+`;
+
+const RowContent = styled.View`
+    flexDirection: row;
+    alignContent: center;
+    alignItems: center;
+    justifyContent: space-evenly;
+`;
+
+const ExpandableCycleCard = (props) => {
+  const [showDetailed, setshowDetailed] = React.useState(false);
+  const [icon, setIcon] = React.useState('chevron-down');
+
+  const {
+    name, muscleGroups, color, exercises,
+  } = props;
+  const FullBody = styled.View`
+        width: 90%;
+        margin: auto;
+        background-color: ${color};
+        borderRadius: 10px;
+        box-shadow: 1px 5px 2px gray;
+        marginBottom: 10px;
+        marginTop: 10px;
+        padding: 10px;
+    `;
+
+  const ExercisesView = (props) => {
+    const { exercises } = props;
+    return (
+      <View style={{ flexDirection: 'column', justifyContent: 'space-evenly' }}>
+        <RowHeader>
+          <SubTitle>Exercise</SubTitle>
+          <SubTitle>Sets x Reps</SubTitle>
+          <SubTitle>Previous</SubTitle>
+        </RowHeader>
+        <FlatList
+          data={exercises}
+          // TODO: rename to exercise name
+          keyExtractor={(item) => item.id}
+          renderItem={({ item }) => (
+            <RowContent>
+              <WorkoutViewText style={{ marginLeft: 10 }}>{item.id}</WorkoutViewText>
+              {/* TODO: center the sets x reps around the x so double digit numbers dont look bad */}
+              <WorkoutViewText>{`${item.sets.length}x${item.sets[0].reps}`}</WorkoutViewText>
+              <WorkoutViewText>{`${item.sets[0].weight} lbs.`}</WorkoutViewText>
+            </RowContent>
+          )}
+        />
+      </View>
+    );
+  };
+
+  function showDetail() {
+    if (icon == 'chevron-down') {
+      setIcon('chevron-up');
+    } else {
+      setIcon('chevron-down');
+    }
+    setshowDetailed(!showDetailed);
+  }
+
+  return (
+    <FullBody>
```

Comment:
I feel like it makes more since to expand the card if the card is pressed at all, not just if they hit the icon.

Reply:
I asked you guys that when I was working on that branch and I thought we agreed on just the icon. I can go and change it if we wanna do that tho lol

Reply:
lets see what the group thinks when we meet